src/components/KRadio/KRadio.vue

```diff
@@ -10,106 +10,196 @@
       type="radio"
       @click="handleClick"
     >
-    <span
+    <div
+      v-if="type === 'card' && $slots.before"
+      class="k-radio-content-before"
+    >
+      <slot name="before" />
+    </div>
+    <template
       v-if="hasLabel"
-      class="k-radio-label"
     >
-      <slot>{{ label }}</slot>
-    </span>
+      <template v-if="type === 'default'">
+        <span class="k-radio-label">
+          <slot>{{ label }}</slot>
+        </span>
+      </template>
+      <template v-if="type === 'card'">
+        <span class="k-radio-label">
+          {{ label }}
+        </span>
+      </template>
```

Comment:
question: Why does the HTML here need to be different based on the `type` property? Can't they utilize the same structure either way?

Reply:
Yeah you're right, I think tried to achieve something else but forgot to change it back